### PR TITLE
Switch form to Screendoor

### PR DESCRIPTION
**Pull request by akegan**
Switching form to screendoor. We'll have just one login, but it seems way better than what we had before in terms of being able to process applications

https://screendoor.dobt.co/san-francisco-digital-services/san-francisco-digital-services/admin/responses?page=1&status=Incoming

@Styler415 I tried to make it look OK, but I'm not sure it's up to snuff. Check the mailing list contact section too to make sure it's OK.

### CODE REVIEW

**Review by aekong — COMMENTED** [review]
this looks ok to me, at least from the code/directory side.  approval pending @Styler415 visual design review.

**Comment by akegan** [comment]
I can also move email and name into two separate fields if that's easier

**Comment by Styler415** [comment]
My only feedback is:
 - **email** field should be on its own line
 - Change **Name** field to **First Name**
 - Change current **email** field to **Last Name**

**Comment by akegan** [comment]
bad news, the name and email on one line thing is a built-in screendoor thing, and if we remove it and replace it with other fields then the forms come in anonymous ☹️

**Comment by akegan** [comment]
Updated to fix the font color issue with the subscribe button for the mailing list, narrowed down the css a bit to not affect all inputs. Ready for review again @Styler415

**Comment by akegan** [comment]
I just also added a resume upload option through screendoor, screenshot posted in recruiting slack channel

**Comment by akegan** [comment]
@aekong or @Styler415 can you do a final QA before we merge this in? Please check the mailchimp subscribe at the bottom of the main page too

File: themes/digitalservices/layouts/partials/header.html
Comment (jszwedko) [inline themes/digitalservices/layouts/partials/header.html]: I believe it's generally preferable to put `<script>`s at the bottom of the `<body>` so it doesn't hold up page loading.

Comment (akegan) [reply themes/digitalservices/layouts/partials/header.html]: hm, screendoor suggests adding it to the `<head>`

**Comment by aekong** [comment]
as far as functionality goes, everything seems to check out.  Only thing I noticed, and it's really small, is the indentation of the text in the screenshot below.  But not sure if that's something you have control over.
screen shot 2018-08-07 at 10 22 15 am

**Comment by akegan** [comment]
fixed @aekong

**Comment by akegan** [comment]
@jszwedko moved script to body in header.html page. Not sure if that's the right place. let me know what you think!

File: themes/digitalservices/layouts/partials/header.html
Comment (jszwedko) [inline themes/digitalservices/layouts/partials/header.html]: You won't want to add `</body>` here as this partial expects there to be more content after. I'm actually a little surprised to see the beginning `<body>` tag in this file. I would have expected it in the layout. If you don't feel like refactoring that bit, the simplest thing to do would be to move it to the `footer.html` partial. Also, I believe you want to leave the CSS here, just move the JS down.

Comment (jszwedko) [reply themes/digitalservices/layouts/partials/header.html]: On a different note ` <link rel="stylesheet" href="/css/styles.css">` is already in this file so you don't need to add it again 😄

File: themes/digitalservices/layouts/partials/header.html
Comment (jszwedko) [inline themes/digitalservices/layouts/partials/header.html]: Since we know we are including jQuery already, you can probably drop this line.

File: themes/digitalservices/layouts/partials/header.html
Comment (aekong) [reply themes/digitalservices/layouts/partials/header.html]: in @akegan's defense - that wasn't her.  it was me! 😢 .  when i tried to fix the asset references.  see https://github.com/SFDigitalServices/sfdigitalservices.github.io/commit/1fcc9643415d189a135995ddb4aadb70537d5c7b#diff-abdd46c1512134cf4cd13003be774c2f

I removed the filter function and forgot to remove the dup

Comment (jszwedko) [reply themes/digitalservices/layouts/partials/header.html]: It's all good, I know it wasn't part of this change set, but just noticed the comment too.

**Review by aekong — APPROVED** [review]
approved! squash and merge!

**Pull request merged**
